cassandradb/seed/analysts.js

```diff
@@ -0,0 +1,32 @@
+const csvWriter = require('csv-write-stream');
+const fs = require('fs');
+var writer = csvWriter();
+const faker = require('faker');
+var counter = 0;
+
+let range = (start, stop, step) =>Array.from({ length: (stop - start) / step + 1}, (_, i) => start + (i * step));
+let years = range(2019, 1995, -1);
+const analystsGen = () => {
+  writer.pipe(fs.createWriteStream('analysts.csv'));
+  for (var i = 1; i < 100; i++) {
+    let name = faker.name.firstName() + ' ' + faker.name.lastName();
+    if (name.includes("'")) {
+      name = name.replace(/'/g, '\'\'');
+    };
+    let year = years[Math.floor(Math.random()*years.length)];
+    let company = faker.company.companyName();
+      if (company.includes("'")) {
+        company = company.replace(/'/g, '\'\'');
+      };
+    writer.write({
+      id: counter++,
+      name: name,
+      year: year,
+      company: company
+    })
+  }
+  writer.end();
```

Comment:
did you encounter memory errors if trying to write large amount of records?   if so, you could look into adding a listener for the 'drain' event on the writer to manage write requests

Reply:
i'm guessing the csvwriter is handling this, so check it out if you're curious